openhantek/src/hantekdso/dsocontrol.cpp

```diff
@@ -303,10 +305,10 @@ DsoControl::BestSamplerateResult DsoControl::computeBestSamplerate(double sample
 }
 
 unsigned DsoControl::updateSamplerate(unsigned downsampler, bool fastRate) {
-    fastRate |= m_specification->supportsFastRate;
```

Comment:
Looks like a broken logic here. If device does not support the fast rate mode, how can we force it?